### ISSUE TITLE
BT-17: permissions codelist mapping table?

### ISSUE THREAD

**Issue by odscjen** [open-contracting/european-union-support#177]
BT-17-Lot (submission electronic) uses the [permission authority table](https://op.europa.eu/en/web/eu-vocabularies/concept-scheme/-/resource?uri=http://publications.europa.eu/resource/authority/permission). Currently the guidance.yaml is:

> [Get the lot for the ProcurementProjectLot](operations.md#get-the-lot-for-a-procurementprojectlot), and set its `.submissionTerms.electronicSubmissionPolicy` according to [the allowed values](https://extensions.open-contracting.org/en/extensions/submissionTerms/master/codelists/).

But not all the codes in the authority table are not a perfect match to the OCDS codelist referenced, namely eforms uses "not-allowed" and OCDS uses "notAllowed" (the other 2 are the same, "allowed" and "required").

Do we need a mapping table for this codelist? Or as it's only 3 options should we put the mapping directly into the guidance?

Additionally the OCDS example is

```json
{"tender":{"lots":[{"submissionMethod":"electronicSubmission","submissionTerms":{"electronicSubmissionPolicy":"allowed"}}]}}
```

but the guidance doesn't mention `submissionMethod` and I notice that this field will be deprecated with 1.2. Should we update the guidance to include this field, or remove it from the example?

@jpmckinney

**Comment by jpmckinney** [comment]
Let's remove `submissionMethod`. The proposal in https://github.com/open-contracting/standard/issues/1183 is for `electronicSubmissionPolicy` to replace `"submissionMethod":"electronicSubmission"`.

I think "set its .submissionTerms.electronicSubmissionPolicy according to [the allowed values](https://extensions.open-contracting.org/en/extensions/submissionTerms/master/codelists/)" is clear enough - an implementer can see that `notAllowed` is the code in OCDS, and it's obvious that `not-allowed` matches `notAllowed`.

**Comment by jpmckinney** [comment]
To avoid repetition of "allowed" in different contexts, maybe reword as

```md
set its `.submissionTerms.electronicSubmissionPolicy` to the corresponding code in the [permission](https://extensions.open-contracting.org/en/extensions/submissionTerms/master/codelists/#permission.csv) codelist.
```
